sonoff/sonoff.h

```diff
@@ -236,7 +236,8 @@ enum Ws2812Color { WS_RED, WS_GREEN, WS_BLUE };
 
 enum LightSubtypes { LST_NONE, LST_SINGLE, LST_COLDWARM, LST_RGB,   LST_RGBW, LST_RGBWC };                   // Do not insert new fields
 enum LightTypes    { LT_BASIC, LT_PWM1,    LT_PWM2,      LT_PWM3,   LT_PWM4,  LT_PWM5,  LT_PWM6, LT_PWM7,
-                     LT_NU8,   LT_SERIAL1, LT_SERIAL2,   LT_WS2812, LT_RGBW,  LT_RGBWC, LT_NU14, LT_NU15 };  // Do not insert new fields
+                     LT_NU8,   LT_SERIAL1, LT_SERIAL2,   LT_WS2812, LT_NEOTRELLIS, LT_RGBW,  LT_RGBWC,
+                     LT_NU14, LT_NU15 };  // Do not insert new fields
```

Comment:
I know that I am not supposed to add new fields here, but how do I add support for a new time of light otherwise?